code/obj/item/storage/secure_safe.dm

```diff
@@ -707,6 +708,13 @@
 	/obj/item/chilly_orb, // a thing to confuse people
 	/obj/item/spacecash/thousand = 3)
 
+/obj/item/storage/secure/ssafe/candy_shop
+	configure_mode = 0
+	random_code = 1
+	spawn_contents = list(/obj/item/robot_foodsynthesizer,\
```

Comment:
not a fan of the food synth being available for humans, it's for cyborgs for a reason

Reply:
The food synth is already available as a spacebux purchase.

It's already not that uncommon to see in human hands.

Reply:
If that's not cool, do you have any suggestions for replacements?

Reply:
I could just remove it if it absolutely needs to go, but I'd rather replace it with something similarly food themed.